modules/osdk-multi-arch-building-images.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-multi-arch-support.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-multi-arch-building-images_{context}"]
+= Building a manifest list of the platforms your Operator supports
+
+You can use the `make docker-buildx` command to build a manifest list of the platforms supported by your Operator and operands. A manifest list references specific image manifests for one or more architectures. An image manifest specifies the platforms that an image supports.
+
+For more information, see link:https://specs.opencontainers.org/image-spec/image-index[OpenContainers Image Index Spec] or link:https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list[Image Manifest v2, Schema 2].
+
+[IMPORTANT]
+====
+If your Operator project deploys an application or other workload resources, the following procedure assumes the application's multi-platform images are built during the application release process.
+====
+
+.Prerequisites
+
+* An Operator project built using the Operator SDK version {osdk_ver} or later
+* Docker installed
+
+.Procedure
+
+. Inspect the image manifests of your Operator and operands to find which platforms your Operator project can support. Run the following command to inspect an image manifest:
++
+[source,terminal]
+----
+$ docker manifest inspect <image_manifest> <1>
+----
+<1> Specifies an image manifest, such as `redhat/ubi9:latest`.
++
+The platforms that your Operator and operands mutually support determine the platform compatibility of your Operator project.
++
+.Example output
+[source,json]
+----
+{
+    "manifests": [
+	{
+	    "digest": "sha256:c0669ef34cdc14332c0f1ab0c2c01acb91d96014b172f1a76f3a39e63d1f0bda",
+	    "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+	    "platform": {
+		"architecture": "amd64",
+		"os": "linux"
+	    },
+	    "size": 528
+	},
+...
+	{
+	    "digest": "sha256:30e6d35703c578ee703230b9dc87ada2ba958c1928615ac8a674fcbbcbb0f281",
+	    "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+	    "platform": {
+		"architecture": "arm64",
+		"os": "linux",
+		"variant": "v8"
+	    },
+	    "size": 528
+	},
+...
+----
+
+** If the previous command does not output platform information, then the specified base image might be a single image instead of an image manifest. You can find which architectures an image supports by running the following command:
```

Comment:
So, a single substep is not sequential, So, since there isn't a second step, it would be a bullet in this case. Just like a procedure with one step. That said, I have gone back and forth as to whether this should be inline or a substep. I will move inline for clarity.